c1: 2
c2: 1
sigma: 1 #en metros
sigma2: 1.3 #en radianes
sigma3: 2
odom_topic: "/odometry/filtered"
obst_topic: "/obstacles"
theta_topic: "/theta_escape"
theta_topic_viz: "/theta_escape_viz"
frame_id: "odom"
control_rate: 100
update_params_service: "/update_parameters"
